Formula/e2fsprogs.rb

```diff
@@ -30,11 +37,14 @@ class E2fsprogs < Formula
   keg_only "this installs several executables which shadow macOS system commands"
```

Comment:
Would be nice to remove keg-only on Linux, but still has duplicates in `krb5` due to internal copy I mentioned in https://github.com/Homebrew/homebrew-core/pull/111319#issuecomment-1281647009